src/rest-server/docs/swagger.yaml

```diff
@@ -1871,52 +1618,59 @@ components:
           type: string
           enum: ['RUNNING', 'STOPPED', 'DRAINING']
           description: RUNNING -> vc is enabled, STOPPED -> vc is disabled, without either new job or running job, DRAINING -> intermedia state from RUNNING to STOPPED, in waiting on existing job.
-    Storage:
+    StorageSummary:
       type: object
       properties:
-        spn:
+        storages:
+          type: array
+          items:
+            type: object
+            properties:
+              name:
+                type: string
+              share:
+                type: boolean
+              volumeName:
+                type: string
+            required: [name, share, volumeName]
+      required: [storages]
+    StorageDetail:
+      type: object
+      properties:
+        name:
+          type: string
+        share:
+          type: boolean
+        volumeName:
           type: string
         type:
           type: string
+          enum:
+          - nfs
+          - samba
+          - azureFile
+          - azureBlob
+          - other
```

Comment:
what is the difference of `other` and `unknown`

Reply:
in current implementation, there're three three types of persistent volume: `nfs`, `azureFile`, `flexVolume`. `flexVolume` includes `samba`, `azureBlob`, and **`other`**. Other types of persistent volume (e.g. `azureDisk`, `cephFS` etc.) are treated as **`unknown`**.

just to distinguish different k8s pv, both `other` and `unknown` do not have extra server information in our api, but could still be used in jobs.
we can also merge them into one type in api.

Reply:
got it. thanks, and I think it is good idea to keep `other` and `unknown`.